cve_bin_tool/strings.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+"""
+These are the customized strings and file classes, by doing this
+the tool is able to support other operating systems like Windows
+and MacOS.
+"""
+
+import sys
+
+class Strings(object):
+
+    def __init__(self, filename):
+        self.filename = filename
+        self.o = ''
+
+    def parse(self):
+        if sys.version_info.major == 3:
+            return self.parse_3()
+        else:
+            return self.parse_2()
```

Comment:
What's the reason for two different implementations?

Reply:
The reason is that I firstly tried to use `open` in both python2 and python3, python2 would not work under python3's code. Thus I realized they have to be implemented with different code (just very likely, I will try more times), and I used `file` in python2.